archive/har.d

```diff
@@ -1,3 +1,23 @@
+/**
+HAR - Human Archive Format
+
+https://github.com/marler8997/har
+
+HAR is a simple format to represent multiple files in a single block of text, i.e.
+---
```

Comment:
why not use ``` instead of --- for code delimiter to avoid clashing w HAR:

```
--- main.d
etc...
```

Reply:
It doesn't clash, except visually.  Does ddoc support triple backticks?

Reply:
I get a ddoc error when using triple backticks